analyses/SC_cd_2010/doc_SC_cd_2010.md

```diff
@@ -0,0 +1,23 @@
+# 2010 South Carolina Congressional Districts
+
+## Redistricting requirements
+In South Carolina, districts must:
+
+1. be contiguous
+1. have equal populations
+1. be geographically compact
+1. preserve county and municipality boundaries as much as possible
+
```

Comment:
I would also note that 2010 was before the Shelby County court ruling so South Carolina's plans had to pass pre-clearance from the DOJ.  I don't think we'd change the simulation per se, but good to note.